install_config/configuring_authentication.adoc

```diff
@@ -200,6 +200,49 @@ identity providers are configured that identify the same set of users and map to
 the same user names.
 |===
 
+[[LookupMappingMethod]]
+==== Lookup Mapping Method
+
+When using the `lookup` mapping method, user provisioning is done by an external system, via the API.
+Typically, identities are automatically mapped to users during login. The 'lookup' mapping method automatically
+disables this automatic mapping, which requires you to map an identity and a user manually.
+
+For more information on identity objects, see the xref:../architecture/additional_concepts/other_api_objects.adoc#identity[Identity]
+user API obejct.
+
+If you are using an identity provider using the `lookup` mapping method, use the following steps for each user after configuring
+the identity provider, as shown below:
+
+. Create an {product-title} User:
++
+----
+$ oc create user <username>
+----
```

Comment:
Also, should the above be more like "Create the specific user, if not created already"? Do you have to create a new user?